docs/data-sources/service_accounts.md

```diff
@@ -0,0 +1,43 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_service_accounts Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Services offer a way to provide programmatic, centrally-controlled, and consistent access controls. For more information, see Twingate's documentation https://www.twingate.com/docs/services.
+---
+
+# twingate_service_accounts (Data Source)
+
+Services offer a way to provide programmatic, centrally-controlled, and consistent access controls. For more information, see Twingate's [documentation](https://www.twingate.com/docs/services).
+
+## Example Usage
+
+```terraform
+data "twingate_service_accounts" "foo" {
+  name = "<your service account's name>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `name` (String) Filter results by the name of the service account.
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+- `services` (Block List) List of Services (see [below for nested schema](#nestedblock--services))
```

Comment:
We should call this value `service_accounts` for consistency

Reply:
updated